geofileops/util/_geoops_ogr.py

```diff
@@ -123,13 +124,8 @@ def _run_ogr(
     start_time = datetime.now()
     if input_layer is None:
         input_layer = gfo.get_only_layer(input_path)
-    if output_path.exists():
-        if force:
-            gfo.remove(output_path)
-        else:
-            logger.info(f"Stop, output exists already {output_path}")
-            return True
-
+    if _io_util.output_exists(path=output_path, force=force):
+        gfo.remove(output_path)
```

Comment:
```suggestion
    if _io_util.output_exists(path=output_path, force=force):
        return True
```

The remove should already have happened in `output_exists()`, and if the output exists already, we can return